static/app/components/onboarding/productSelection.tsx

```diff
@@ -159,32 +217,62 @@ export function ProductSelection({
             label={t('Performance Monitoring')}
           />
         </Tooltip>
-        <Tooltip
-          title={
-            sessionReplayProductDisabled?.reason ?? (
-              <TooltipDescription>
-                {t(
-                  'Video-like reproductions of user sessions with debugging context to help you confirm issue impact and troubleshoot faster.'
-                )}
-                <ExternalLink href="https://docs.sentry.io/platforms/javascript/guides/react/session-replay/">
-                  {t('Read the Docs')}
-                </ExternalLink>
-              </TooltipDescription>
-            )
-          }
-          isHoverable
-        >
-          <Product
-            onClick={
-              sessionReplayProductDisabled
-                ? sessionReplayProductDisabled?.onClick
-                : () => handleClickProduct(ProductSolution.SESSION_REPLAY)
+        {!hideSessionReplay && (
+          <Tooltip
+            title={
+              sessionReplayProductDisabled?.reason ?? (
+                <TooltipDescription>
+                  {t(
+                    'Video-like reproductions of user sessions with debugging context to help you confirm issue impact and troubleshoot faster.'
+                  )}
+                  <ExternalLink href="https://docs.sentry.io/platforms/javascript/guides/react/session-replay/">
+                    {t('Read the Docs')}
+                  </ExternalLink>
+                </TooltipDescription>
+              )
             }
-            disabled={!!sessionReplayProductDisabled}
-            checked={products.includes(ProductSolution.SESSION_REPLAY)}
-            label={t('Session Replay')}
-          />
-        </Tooltip>
+            isHoverable
+          >
+            <Product
+              onClick={
+                sessionReplayProductDisabled
+                  ? sessionReplayProductDisabled?.onClick
+                  : () => handleClickProduct(ProductSolution.SESSION_REPLAY)
+              }
+              disabled={!!sessionReplayProductDisabled}
+              checked={products.includes(ProductSolution.SESSION_REPLAY)}
+              label={t('Session Replay')}
+            />
+          </Tooltip>
+        )}
+        {includeProfiling && (
+          <Tooltip
+            title={
+              profilingProductDisabled?.reason ?? (
+                <TooltipDescription>
+                  {t(
+                    'Enable code-level visibility in production environments, helping you swiftly detect performance bottlenecks and visualize hot paths in your code. Requires Performance Monitoring.'
+                  )}
+                  <ExternalLink href="https://docs.sentry.io/platforms/python/profiling/">
+                    {t('Read the Docs')}
+                  </ExternalLink>
```

Comment:
we currently don't have documentation for profiling exclusively for Django. Shall we add it?

Reply:
Nah, I think linking to the python generic docs is sufficient 👍